src/data/uiConfig.ts

```diff
@@ -25,6 +25,13 @@ export const isTokenAssets = (config: ConfigProps) => {
   return config.cellarNameKey === CellarNameKey.AAVE
 }
 
+export const IsOneMonthEthBtc = (config: ConfigProps) => {
+  return (
+    config.cellarNameKey === CellarNameKey.ETH_BTC_MOM ||
+    config.cellarNameKey === CellarNameKey.ETH_BTC_TREND
+  )
+}
+
```

Comment:
I suggest it to be like this
```suggestion
export const intervalGainTimeline = (config: ConfigProps) => {
if(config.cellarNameKey === CellarNameKey.ETH_BTC_MOM ||
    config.cellarNameKey === CellarNameKey.ETH_BTC_TREND){
    return "monthly"
    }
 return "weekly"
}
```
and put it on `PageCellar.tsx` line 64

Reply:
note: use lowercase for variable